packages/desktop-client/src/components/filters/FiltersMenu.jsx

```diff
@@ -594,7 +594,13 @@ export function AppliedFilters({
   onDelete,
   conditionsOp,
   onCondOpChange,
+  onUpdateChange,
 }) {
+  const onUpdateAppliedFilter = (filter, newFilter) => {
+    onUpdateChange?.(null, 'modify');
```

Comment:
Is the null argument here correct? I think onReportChange is still only taking one arg?

Reply:
updated